help/set_ram.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Set maximum RAM"
+permalink: /help/set-ram/
+---
+(Valid for TripleA v1.9.0.0.5401+)<br>
+It may happen that your TripleA client exits with an `OutOfMemoryError`, even if your PC has more RAM to offer.<br>
+In this case you might want to increase the usable amount of RAM for your installation of TripleA.<br>
+There are multiple ways to achieve this, and they all depend on the way your version of TripleA has been installed on your computer.
```

Comment:
IMO keep it simple and provide instructions for just the latest and via installer  -  avoid giving options.